### PR TITLE
Update to use ilib-lint-webos plugin for html-formatter

### PR DESCRIPTION
**Pull request by gouniLee**
Update to use ilib-lint-webos plugin for html-formatter

### CODE REVIEW

File: lint-webos/webos-result.html [hunk 1]
```diff
@@ -0,0 +1,427 @@
+<!DOCTYPE html>
```
Comment (ehoogerbeets) [inline lint-webos/webos-result.html:1]: Should this file be checked in? Shouldn't this be generated each time as a demo to the user?

Comment (gouniLee) [reply lint-webos/webos-result.html:1]: it shouldn't be here. I just added it temporarily to show what the HTML output file looks like at thie moment, and I prefer the summary to come first and then the results list. I'm assuming the results list is very large as the webOS file is very large.

**Review by ehoogerbeets — COMMENTED** [review]
Probably not a good idea to have hard-coded paths on your own machine

**Comment by gouniLee** [comment]
> Probably not a good idea to have hard-coded paths on your own machine

Yeah, I agree. currently, it's a middle of testing. I'll update it when all of the related changes are published.

**Comment by gouniLee** [comment]
@ehoogerbeets It is ready to get the review again.

**Review by ehoogerbeets — APPROVED** [review]

**Pull request merged**